src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,105 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The source of the logo for the token.
+     */
+    logoSrc?: string;
+    /**
+     * The name of the Token.
+     */
+    tokenName?: string;
+    /**
+     * The symbol of the Token.
+     */
+    symbol?: string;
+    /**
+     * The amount of the Token.
+     */
+    amount?: number | string;
+    /**
+     * The price of the Token.
+     */
+    USDAmount?: number | string;
+    /**
+     * changed price amount (E.g. in last 24h).
+     */
+    changedAmount?: number | string;
+    /**
+     * changed price amount ratio (E.g. in last 24h).
+     */
+    changedPercentage?: number | string;
```

Comment:
Similar here, I'd only have a `priceChange` property that refers to the price change in percentage of the asset and calculate the amount change on the component like:
```
const usdAmount = amount * usdPrice;
const oldUsdAmount = (100 / (priceChange + 100)) * usdAmount;
const usdAmountChange = usdAmount - oldUsdAmount;
```